src/store/tracks.js

```diff
@@ -47,6 +47,24 @@ export default {
           return Promise.resolve(false);
         });
     },
+    read({ commit, rootState }, id) {
+      return read(rootState.auth, id)
+        .then((result) => {
+          commit("setTrack", { id: result.id, track: result });
+          return Promise.resolve(result.id);
+        })
+        .catch((error) => {
+          this.commit("addError", error);
+          return Promise.resolve(false);
+        });
+    },
+    readMultiple({ dispatch }, ids) {
+      const promises = [];
+      ids.forEach((id) => {
+        promises.push(dispatch("read", id));
+      });
+      return Promise.all(promises);
```

Comment:
I think we will need to handle these requests sequentially. This could potentially launch thousands of requests at the same time.

Reply:
We don't want to launch that many requests period.
One of the reasons why I created accentor/api#39

Reply:
We could also get all the tracks in the scope of the genre that is merged into. This means fetching more data than we stricly need, but it means that we can keep the number of requests low.
(That would still not solve the problem after a destroy, though)